discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -66,7 +66,7 @@ def shutdown_all_nodes(self):
         for node in nodes:
             self.shutdown_node(node.name())
 
-    def start_node(self, node_name, check_ips):
+    def start_node(self, node_name, check_ips=True):
```

Comment:
Not sure about this change... @yevgeny-shnaidman please verify this - it looks like you introduced the parameter. Does it make sense for it to be true by default (looks like the previous behavior, before the parameter, was if it was "True")?

Reply:
PR that introduced this change: #424

Not sure how this didn't come up until now... This is the line that makes use of the old signature: https://github.com/openshift/assisted-test-infra/blob/d3bf8bed79cc68f1fed0af52350fb86a0750ff67/discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py#L91